cl/corpus_importer/management/commands/make_aws_manifest_from_search.py

```diff
@@ -0,0 +1,186 @@
+import csv
+import io
+import operator
+from functools import reduce
+from typing import Any
+
+import boto3
+from django.core.management import CommandParser  # type: ignore
+from django_elasticsearch_dsl.search import Search
+from elasticsearch_dsl import Q
+
+from cl.lib.argparse_types import valid_date
+from cl.lib.command_utils import VerboseCommand
+from cl.lib.elasticsearch_utils import (
+    build_daterange_query,
+    build_fulltext_query,
+    build_sort_results,
+)
+from cl.search.constants import SEARCH_OPINION_QUERY_FIELDS
+from cl.search.documents import OpinionDocument
+from cl.search.models import SEARCH_TYPES
+
+s3_client = boto3.client("s3")
+
+
+def build_base_query(options: dict[str, Any]) -> Search:
+    """
+    Constructs a base Full text Elasticsearch query using the provided options.
+
+    Args:
+        options (dict[str, Any]): A dictionary containing search options:
+            - search_text (str): The text to search for.
+            - search_type (str): The type of search.
+            - filed_after (date | str): Filter documents filed **on or after**
+            the specified date.
+            - filed_before (date | str): Filter documents filed **on or before**
+            the specified date.
+            - es_page_size (int): The desired number of results per page.
+
+    Returns:
+        Search: The constructed Elasticsearch query object.
+
+    Raises:
+        NotImplementedError: If the provided search_type is not supported.
+    """
+    cd = {
+        "q": options["search_text"],
+        "type": options["search_type"],
+    }
+    match options["search_type"]:
+        case SEARCH_TYPES.OPINION:
+            unique_sorting_key = "id"
+            cd["order_by"] = "score desc"
```

Comment:
Since we don't care about the scores or the order of results, we can remove this.

Reply:
Thanks for your suggestion! I'll remove this.